utils/common.go

```diff
@@ -90,3 +92,44 @@ func Compare(a, b []byte) bool {
 	}
 	return c == 0
 }
+
+func GetExternalsVolumesAndVolumesMount(externalResourceConfigMaps map[string]v1.ConfigMap, prefixMountPathMap map[string]string) ([]v1.Volume, []v1.VolumeMount) {
+	volumes := make([]v1.Volume, 0)
+	volumeMounts := make([]v1.VolumeMount, 0)
+	for keycm, cm := range externalResourceConfigMaps {
```

Comment:
```suggestion
	for keycm, cm := range configMaps {
```

Reply:
This is a generic struct constructor. `externalResources` means nothing in this context.